test/REQUIRE

```diff
@@ -0,0 +1 @@
+SpecialFunctions
```

Comment:
I think we don't need this, since SpecialFunctions is already in the REQUIRE of the package

Reply:
Got it.